tests/integration/welcome.test.js

```diff
@@ -0,0 +1,15 @@
+const {startServer,stopServer}=require('../../src/server')
+ const fetch = require('node-fetch')
+
+ describe('test welcome page',()=>{
+     beforeAll(async ()=>{
+         await startServer(20200)
```

Comment:
I actually cannot use `20200` on my computer. Could you make this configurable?

```
const testPort = process.env.TEST_PORT || 20200
const baseUrl = `http://localhost:${testPort}`
...
await startServer( testPort )
...
await fetch(baseUrl)
...
```

Reply:
Ok